addons/binding/org.openhab.binding.lametrictime/ESH-INF/thing/channels.xml

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="lametrictime"
+                          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                          xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+                          xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+    <channel-type id="jsonNotificationType">
+        <item-type>String</item-type>
+        <label>Advanced Notification</label>
+        <description>Channel to send custom messages with the LaMetric APIs JSON structure.</description>
+    </channel-type>
+
+    <channel-type id="simpleNotificationType">
+        <item-type>String</item-type>
+        <label>Simple notification</label>
+        <description>Send a notification to LaMetric Time</description>
+        <category>Recorder</category>
+    </channel-type>
+
+    <channel-type id="brightnessType">
+        <item-type>Dimmer</item-type>
+        <label>Display Brightness</label>
+        <description>Adjust the brightness of the display</description>
+        <state min="0" max="100" step="1" readOnly="false"/>
+    </channel-type>
+
+    <channel-type id="brightnessModeType">
+        <item-type>String</item-type>
+        <label>Display Brightness Mode</label>
+        <description>Manual or Auto mode</description>
+        <state>
+            <options>
+                <option value="auto">Automatic</option>
+                <option value="manual">Manual</option>
+            </options>
+        </state>
+    </channel-type>
+
+    <channel-type id="volumeType">
+        <item-type>Dimmer</item-type>
+        <label>Volume</label>
+        <description>Adjust the volume of the speaker</description>
+    </channel-type>
+
+    <channel-type id="bluetoothActiveType">
+        <item-type>Switch</item-type>
+        <label>Bluetooth</label>
+        <description>Switch Bluetooth on/off</description>
+    </channel-type>
+
+    <channel-type id="appType">
+        <item-type>String</item-type>
+        <label>Application</label>
+        <description>Active application</description>
+        <state>
```

Comment:
I tested out removing this, but it causes the Paper UI to fail to render the item as a selection. With the placeholder in there, everything works as expected. Perhaps there is a timing issue here. This might require some additional investigation if we really want to remove it, but it doesn't seem like a big deal to leave it.

Reply:
I think there must be some timing issue. I see this effect from time to time even if this xml is in place (in that case it shows the invalid option "App", which isn't imho any better, so I'd still vote for removing this part here).

Reply:
Ok; I'll remove it.